src/Symfony/Component/HttpClient/TraceableHttpClient.php

```diff
@@ -50,7 +51,10 @@ public function request(string $method, string $url, array $options = []): Respo
             }
         };
 
-        return $this->client->request($method, $url, $options);
+        $response = $this->client->request($method, $url, $options);
+        $body = $response->toArray();
```

Comment:
this defeats lazyness i believe ... shouldnt we show the body "if requested" during the app's lifecycle? If the request was never made, the profiler should display it as such IMHO and not perform the actual request.

Reply:
Correct. This is not easy to achieve with the current available infrastructure. I'd suggest postponing this part for later.

Reply:
Ok I removed this part from PR/Issue but I'm curious to know what's your idea or the expected behavior ? A button that display (= retrieve) body only if clicked

Reply:
The body should be caught while it is being downloaded, then just displayed here. Monitoring downloads requires some code infrastructure that's not available for now.